src/main/webapp/tab.html

```diff
@@ -11,7 +11,15 @@
     <div class = "wrap">
         <div class = "navigationBar" id="nav_bar"></div>
         <div class = "bodyMain" id= "body_main"></div>
-            <div class = "leftCol" id = "left_col"></div>
+            <div class = "leftCol" id = "left_col">
+                <div id="mySidebar" class="sidebar">
+                <p class="rep-name">null</p>
+                <a href = "/repList.html" id="repRetButton"><span><i class="material-icons">list</i><span class="repList-text icon-text">Other Representatives</span></a><br>
+                <a href="#" value="Education" id="eduButton" onclick="getTab(this.value)"><span><i class="material-icons">school</i><span class="edu-text icon-text">Education</span></a><br>
```

Comment:
These should no longer be hardcoded as tabs are dynamically created - might be better to abstract into a function